website/content/docs/discovery/dns.mdx

```diff
@@ -285,6 +285,50 @@ caching of DNS results. However, there are many situations in which caching is
 desirable for performance and scalability. This is discussed more in the tutorial
 for [DNS caching](https://learn.hashicorp.com/tutorials/consul/dns-caching).
 
+## DNS with ACLs
```

Comment:
The majority of this page highlights features provided by the DNS interface, whereas this section explains how DNS interacts with another area of Consul – the ACL system. What do you think about moving this to the last section of the page – after the feature sections – or to a separate page altogether?

Reply:
I don't see a great place for a separate page in the nav:
![image](https://user-images.githubusercontent.com/85913323/132780491-20ce7476-ebd0-4881-baa0-a837b96bd5f9.png)

I'm fine with moving this content to the bottom of the page. I thought I recall us including use with ACLs at the bottom of another page, but I can't find it at the moment. I think that's our best option.

Long-term, I think it's a good idea to help users understand how the pieces fit together better (e.g., Learn tutorials that give guidance on how to accomplish the same thing with ACLs, mentioning use with ACLs in the docs, etc).